tests/test_isolation_macos.sh

```
@@ -368,6 +382,95 @@ else
+  "${FAKE_HOME}/.claude.jsonEVIL"
+  "${FAKE_HOME}/.claude.json.tmp.1.2.extra"
+  "${FAKE_HOME}/.claude.json.tmp.abc.1"
+  "${FAKE_HOME}/.claude.json.tmp.1."
+)
+test12_all_ok=1
+for deny_path in "${deny_paths[@]}"; do
+  rm -f "$deny_path"
+  status=0
+  (cd "$FAKE_WORKTREE" && run_shim VIBECORP_ISOLATION=1 -- write-path "$deny_path") || status=$?
+  if [[ "$status" -ne 0 && ! -f "$deny_path" ]]; then
+    pass "範囲外パス拒否: ${deny_path##${FAKE_HOME}/}"
```

Shellcheck (0.11.0)

[info] Expansions inside ${..} need to be quoted separately, otherwise they match as patterns.

(SC2295)